pkgs/tools/networking/openssh/common.nix

```diff
@@ -98,6 +82,8 @@ stdenv.mkDerivation rec {
     ++ optional (!linkOpenssl) "--without-openssl"
     ++ extraConfigureFlags;
 
+  ${if stdenv.hostPlatform.isStatic then "NIX_LDFLAGS" else null}="-lkeyutils -laudit";
+
```

Comment:
Nice. Didn't knew about this "null =" trick

Reply:
You need conditional on withKerberos, I think

Reply:
And I thought NIX_LDFLAGS is generally frown upon.

Reply:
> And I thought NIX_LDFLAGS is generally frown upon.

That is correct. For some `LDFLAGS` is not respected here, so I don't see another way.

Reply:
Well, you can sed stuff, like I did in my PR.